.ci/bump-version.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -euxo pipefail
```

Comment:
What's `u` and `o` ?

Reply:
-u: Attempt to use undefined variable outputs error message, and forces an exit
-o pipefail: Causes a pipeline to return the exit status of the last command in the pipe that returned a non-zero return value.

https://www.tldp.org/LDP/abs/html/options.html

Reply:
TIL, thanks! So `-o` is like _fail as soon as any point of the pipe fails_?

Reply:
yep, this is what it does, the following command does not fail because `echo "OK"`does not fails

```
cat noToy|echo "OK"
echo $?
```

if you set `-o pipefail`, the error is propagated

```
set -o pipefail
cat noToy|echo "OK"
echo $?
```